android/res/values-pt-rBR/strings.xml

```diff
@@ -35,6 +35,19 @@
 	<string name="country_status_download_failed">O download falhou</string>
 	<!-- Button text for the button under the country_status_download_failed message -->
 	<string name="try_again">Tentar novamente</string>
+	<string name="about_menu_title">Sobre o Organic Maps</string>
+	<!-- Text in About screen -->
+	<string name="about_headline">Gratuito para todos, feito com amor.</string>
+	<!-- Text in About screen -->
+	<string name="about_proposition_1">• Sem anúncios, sem rastreamento</string>
+	<!-- Text in About screen -->
+	<string name="about_proposition_2">• Não descarrega a bateria</string>
+	<!-- Text in About screen -->
+	<string name="about_proposition_3">• Rápido e minimalista</string>
+	<!-- Text in About screen -->
+	<string name="about_developed_by_enthusiasts">Criado por entusiastas da comunidade de código aberto.</string>
+	<!-- Text in About screen -->
+	<string name="about_report_to_osm">Corrija problemas de dados de mapas em OpenStreetMap.org.</string>
```

Comment:
<string name="about_menu_title">Sobre o Organic Maps</string>
 	<!-- Text in About screen -->
 	<string name="about_headline">Gratuito para todos, feito com amor.</string>
 	<!-- Text in About screen -->
 	<string name="about_proposition_1">• Sem anúncios, rastreamento ou coleta de dados</string>
 	<!-- Text in About screen -->
 	<string name="about_proposition_2">• Baixo consumo de bateria, sem atualização em segundo plano</string>
 	<!-- Text in About screen -->
 	<string name="about_proposition_3">• Rápido, minimalista e sem recursos desnecessários</string>
 	<!-- Text in About screen -->
 	<string name="about_developed_by_enthusiasts">Criado por entusiastas e voluntários com o poder do código aberto.</string>
 	<!-- Text in About screen -->
 	<string name="about_report_to_osm">Favor corrigir todos os problemas do mapa diretamente em OpenStreetMap.org.</string>